make

```diff
@@ -12,66 +12,82 @@
 # Forces the script to exit on error
 set -e
 
+export BOOT_NAME=boot
+export MI_NAME=mi
+
 # Setup environment variable to find standard library
 cd stdlib; export MCORE_STDLIB=`pwd`; cd ..;
 
 # General function for building the project
 build() {
     mkdir -p build
     dune build
-    cp -f _build/install/default/bin/boot.mi build/mi
+    cp -f _build/install/default/bin/boot.mi build/$BOOT_NAME
+    if [ -e build/$MI_NAME ]
+    then
+        echo "Bootstrapped compiler already exists. Run 'make clean' before to recompile. "
+    else
+        echo "Bootstrapping the Miking compiler (1st round, might take a few minutes)"
+        time (build/$BOOT_NAME src/main/mi.mc -- compile src/main/mi.mc)
+        mv -f $MI_NAME build/$MI_NAME
+        echo "Bootstrapping the Miking compiler (2st round, might take some more time)"
```

Comment:
Is there a benefit to bootstrapping the compiler twice?

Reply:
Right now, I would like to do it for testing, both regarding time measurements and that it works.